src/07-projects/projects.apib

```diff
@@ -11,6 +11,14 @@ Get a list of projects.
 + Request (application/json;charset=utf-8)
 
     + Attributes (object)
+        + filter (object, optional)
+            + status (enum[string], optional)
+                + Members
+                    + active
```

Comment:
Isn't active a bit of a weird term? Sounds to me like this is the status during the meeting and that it means the meeting is happening at the moment.

Reply:
You're right, a name like `planning_status` would make more sense here, it has already implemented with `status` for a while but I can easily change it. Any other suggestions?

Reply:
@WouterSioen this is the status of a project, not of a meeting. There are alternatives for `active`, like `busy`, `ongoing`, `current`, but I think `active` is the best one.

Reply:
Then it makes sense, not sure where I got the idea that this was for meetings.